kindlecomicconverter/KCC_gui.py

```diff
@@ -452,12 +452,23 @@ def selectDir(self):
         if self.needClean:
             self.needClean = False
             GUI.jobList.clear()
+        
         dname = QtWidgets.QFileDialog.getExistingDirectory(MW, 'Select directory', self.lastPath)
         if dname != '':
             if sys.platform.startswith('win'):
                 dname = dname.replace('/', '\\')
             self.lastPath = os.path.abspath(os.path.join(dname, os.pardir))
+            
             GUI.jobList.addItem(dname)
+            
+            for root, _, files in os.walk(dname):
+                for file in files:
+                    if file.lower().endswith('.cbz'):
```

Comment:
Only supporting CBZ and not other archives like zip/cbr isn't the best.

Reply:
done.